includes/class-wc-stripe-hook-manager.php

```
@@ -0,0 +1,134 @@
+		return true;
+	}
+
+	/**
+	 * Check if the payment method ID is valid.
+	 *
+	 * @param string $payment_method_id The payment method ID to check.
+	 * @return bool True if the payment method ID is valid, false otherwise.
+	 */
+	public function is_valid_payment_method_id( string $payment_method_id ): bool {
+		return 'stripe' === $payment_method_id || str_starts_with( $payment_method_id, 'stripe_' );
+	}
```

PHPMD (2.15.0)

[error] The method is_valid_payment_method_id is not named in camelCase. (undefined)

(CamelCaseMethodName)